src/backup.rs

```diff
@@ -78,6 +78,60 @@ impl BackupEngine {
             Ok(())
         }
     }
+    /// Restore from the latest backup
```

Comment:
Please change the formatting of this doc-comment.
Markdown will not follow your line feeds, so all these lines would be probably just one paragraph in the docs.
Moreover, you need to surround your example with `\`\`\`ignore` and `\`\`\`` lines.  With `ignore`, your example will _not_ be considered doc-test, so compiler will ignore it. This is not good in terms of actuality of documentation, so I suggest you to try make your doc-example compilable.
See https://doc.rust-lang.org/rustdoc/documentation-tests.html for more information. Also see https://doc.rust-lang.org/rust-by-example/meta/doc.html as an example of good doc-comment.

Thanks

Reply:
Thanks.  I need to spend more time understanding rust documentation. Will take care of this.

Reply:
@vitvakatu thanks for `cargo doc --open`. I helped me to see the doc output.  I have resolved the doc issue now.